dotCMS/src/main/java/com/dotmarketing/viewtools/MailerTool.java

```diff
@@ -50,25 +47,27 @@
 	 */
 	public String sendEmail(String to, String from, String subject,
 			String message, Boolean html) {
-		Session s = null;
+		Session session = null;
+		Context ctx = null;
 		try {
-			Context ctx = (Context) new InitialContext();
-			s = (javax.mail.Session) JNDIUtil.lookup(ctx, "mail/MailSession");
-		} catch (NamingException e1) {
-			Logger.error(this,e1.getMessage(),e1);
-		}
+			ctx = (Context) new InitialContext().lookup("java:comp/env");
+			session = (javax.mail.Session) ctx.lookup("mail/MailSession");
+		} catch (Exception e1) {
+			try {
+				Logger.debug(this, "Using the jndi intitialContext().");
+				ctx = new InitialContext();
+				session = (javax.mail.Session) ctx.lookup("mail/MailSession");
+			} catch (Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)